src/sql/workbench/contrib/notebook/browser/notebook.component.ts

```diff
@@ -232,7 +232,9 @@ export class NotebookComponent extends AngularDisposable implements OnInit, OnDe
 				let cells = [...new Set(oldDecorationsRange.map(item => item.cell))].filter(c => c.cellType === 'markdown');
 				cells.forEach(cell => {
 					let cellOldDecorations = oldDecorationsRange.filter(r => r.cell === cell);
-					let cellEditor = this.cellEditors.find(c => c.cellGuid() === cell.cellGuid);
+					// In split mode we have code and text cells with the same Guid,
```

Comment:
Is there not a way to use the cellType? `hasEditor` is kind of a weird way to go about checking if it's a text cell.

Reply:
Sounds good, thanks.